test/protocols/MevShare.t.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "src/protocols/MevShare.sol";
+
+contract MevShareTest is Test {
+    function testEncodeMevShare() public {
+        MevShare.Bundle memory bundle;
+        bundle.version = "";
+        bundle.inclusionBlock = 1;
+
+        bundle.bodies = new bytes[](1);
+        bundle.bodies[0] = abi.encode(1234);
```

Comment:
just a nit but might change `1234` to `0x1234`. I like to use hex values directly in cases like this where you'd want to see [where] the same number shows up in the test assertion